### DISCUSSION TITLE
Freezes, freezes, freezes

### DISCUSSION THREAD

**Discussion by Cockpitlyrics** [kubilus1/autoortho#178] [General]
I'm a bit surprised that there is hardly any talk about the recurring freezes here. I use AO with XP12 and a 250 MBit/s line and a Ryzen 7 5800X3D. After about 10 minutes, X-Plane freezes for a few minutes. I have actually tried everything: Changed CPU allocation via ProcessLasso, changed image provider, excluded cache and AutoOrtho folder from Windows Defender, reduced texture quality in XP, it all did nothing. XP continues to stop for a few minutes. The problem exists for some users, but for the majority it seems to work without problems. I am at a loss as to what to do. And that's especially a shame since AO is a real must have. I don't even want to fly without it anymore.

**Comment by kubilus1** [comment]
You may want to check the cache directory that's in use.  Is this on a really slow HDD vs and SSD?  Check if Windows Defender is busy scanning the folder (I notice Windows Defender will ignore excludes and keeps kicking itself back on).

**Comment by kubilus1** [comment]
In the screen there is regular status line outputs, can you post one here?

**Comment by lknfly1** [comment]
Same here! Still no solution on my side, I had no choice than to use a HDD (Custom Scenery Folder)  (XP on SSD) and my old ortho files are on this HDD which didn't create such perf issues when I select them on Xorganiser to make a try with Xp12 but for AO it is another story, I even tried Arc/Bing etc and Process Lasso but it didn't change a thing (I still have to try different cores I don't know, I'll see) ... I just can't pinpoint the source of such frozen screens, drop of fps to 5 image per second more than ten times per flight and then it gets back to normal which I don't have that with the original ortho files. I don't want to give up, AO is a Must Have, a Game Changer but I just can't find the solution so far even by reading posts here and there and by trying to follow some instructions it doesn't work.... yeah some of us have issue, others not!

**Comment by gusram5** [comment]
I have moved all the folders and cache to a SSD drive, in XP11 is very smooth without any issues but in XP12 I have a lot of stutters and freezes with a message of  "try to find back up chunk"
